Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -2021,6 +2048,178 @@ public ImageView.ScaleType getScaleTypes(String scale){
+                ImageView imageView = new ImageView(context);
+
+                // Create a circular bitmap drawable
+                BitmapDrawable bitmapDrawable = new BitmapDrawable(context.getResources(), decodedByte);
+                Drawable circularDrawable = createCircularDrawable(bitmapDrawable);
+
+                imageView.setImageDrawable(circularDrawable);
+                imageView.setScaleType(ImageView.ScaleType.FIT_CENTER);
+                imageView.setAdjustViewBounds(true);
+                imageView.setClipToOutline(true);
+                imageView.setRotation(270);
+                layout.removeAllViews();
+                layout.addView(imageView);
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    });
+}
+
+private Drawable createCircularDrawable(BitmapDrawable bitmapDrawable) {
+    Bitmap bitmap = bitmapDrawable.getBitmap();
+    Bitmap circularBitmap = Bitmap.createBitmap(bitmap.getWidth(), bitmap.getHeight(), Bitmap.Config.ARGB_8888);
+    Canvas canvas = new Canvas(circularBitmap);
+    Paint paint = new Paint();
+    paint.setAntiAlias(true);
+
+    // Create a circular shader
+    BitmapShader shader = new BitmapShader(bitmap, Shader.TileMode.CLAMP, Shader.TileMode.CLAMP);
+    Matrix matrix = new Matrix();
+    matrix.setTranslate((bitmap.getWidth() - canvas.getWidth()) / 2f, (bitmap.getHeight() - canvas.getHeight()) / 2f);
+    shader.setLocalMatrix(matrix);
+    paint.setShader(shader);
+
+    // Draw the circular shape onto the canvas
+    float radius = Math.min(bitmap.getWidth(), bitmap.getHeight()) / 2f;
+    canvas.drawCircle(canvas.getWidth() / 2f, canvas.getHeight() / 2f, radius, paint);
+
+    return new BitmapDrawable(context.getResources(), circularBitmap);
+}
+
+
+@JavascriptInterface
+    public void renderCameraProfilePicture(String id) {
+        
+        activity.runOnUiThread(() -> {
+           
+             if (isCameraPermissionGranted()) {
+                 //System.out.println(id);
+            View invoiceLayout = LayoutInflater.from(context).inflate(R.layout.profile_picture_camera_preview, null, false);
+            previewView = invoiceLayout.findViewById(R.id.previewView);
+            bCapture = invoiceLayout.findViewById(R.id.bCapture);
+            bCapture.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    capturePhoto();
+                }
+            });
+            ListenableFuture<ProcessCameraProvider> cameraProviderFuture = ProcessCameraProvider.getInstance(context);
+            cameraProviderFuture.addListener(() -> {
+            try {
+                ProcessCameraProvider cameraProvider = cameraProviderFuture.get();
+                startCameraX(cameraProvider);
+            } catch (ExecutionException | InterruptedException e) {
+                e.printStackTrace();
```

Comment:
If the exception happend handle It properly
like if you are not doing anything in catch block, your next code block will be executed, Please handle that Properly, Here you can simple add a return statement